platform/core/src/services/CustomizationService/CustomizationService.ts

```diff
@@ -131,13 +131,18 @@ export default class CustomizationService extends PubSubService {
       }
     });
 
-    this.addReferences(this.configuration);
+    // This allows us to control when configuration references are added.
```

Comment:
@wayfarer3130 could you please comment on this? Thanks. See also line 136 below.

Reply:
I'd really rather not have the init method control whether the references get added or not.  
The right solution is only to allow the update to occur for a given customization item once.  
What about defining a property on this.configuration indicating that references have been added, so the code becomes
if (!this.configuration?._hasBeenAdded ) {
   this.addReferences(this.configuration);
   Object.defineProperty(this.configuration, "_hasBeenAdded", { value: true });
}

It might be desirable to do that on global/default customizations as well.

Reply:
@wayfarer3130 , that is a good suggestion. I don't think we have to do that for global/default customizations because they are cleared at the beginning of this `init` method. Does that make sense?